sys/cloudformation/stack.yaml

```diff
@@ -342,10 +361,11 @@ Resources:
   # ECS TASK DEFINITION
   ecstask:
     Type: AWS::ECS::TaskDefinition
+    DeletionPolicy: Retain
```

Comment:
Both `UpdateReplacePolicy` and `DeletionPolicy` are needed to protect from deletion

Reply:
fixed